solr/solr-ref-guide/src/updating-parts-of-documents.adoc

```diff
@@ -103,6 +103,108 @@ The resulting document in our collection will be:
 }
 ----
 
+=== Updating Child Documents
+
+Solr supports modifying, adding and removing child documents as part of atomic updates. +
+Schema and configuration requirements are detailed in
+<<indexing-nested-documents#schema-configuration, Indexing Nested Documents>>. +
```

Comment:
-- and also updating-parts-of-documents.adoc heading "Field Storage", which I would even list _first_ since what we're describing here fits underneath that conceptually.